app/views/application/confirm-withdraw.njk

```diff
@@ -3,7 +3,7 @@
 {% set application = data.applications[applicationId] %}
 {% set name = [application["personal-details"]["given-name"], application["personal-details"]["family-name"]] | join(" ") %}
 
-{% set title = "Confirm that you want to withdraw the offer" %}
+{% set title = "Confirm offer withdrawal" %}
```

Comment:
How about:
“Are you sure you want to withdraw this offer?”

This matches the language we use on the candidate side:
https://bat-design-history.herokuapp.com/images/apply-for-teacher-training/apply-launch/54-delete-role.png

Reply:
Made a ticket to do this work—thanks @fofr